prow/cluster/jobs/all-periodics.yaml

```diff
@@ -35,9 +35,9 @@ periodics:
   name: daily-performance-benchmark
   branches: master
   decorate: true
-  timeout: 24h
+  timeout: 12h
```

Comment:
I think this is a noop. The `timeout` is read from `decoration_config.timeout`

Reply:
you mean this actually never be used? where is the `decoration_config.timeout`>

Reply:
> you mean this is actually never been used? where is the `decoration_config.timeout`?

Yes, you should only need `decoration_config.timeout`, which is the change on [line 40](https://github.com/istio/test-infra/pull/2534/files#diff-fa4a92fd9ad0bf1866a82628a7317117R40).

Reply:
This is more of an FYI, since it is not an error to have it defined.